be/src/exec/pipeline/fragment_executor.cpp

```diff
@@ -118,7 +118,7 @@ Status FragmentExecutor::prepare(ExecEnv* exec_env, const TExecPlanFragmentParam
         exec_env->runtime_filter_worker()->open_query(query_id, request.query_options, params.runtime_filter_params,
                                                       true);
     }
-    exec_env->stream_mgr()->prepare_pass_through_chunk_buffer(query_id);
+    _query_ctx->prepare_pass_through_chunk_buffer();
```

Comment:
Because each fragment instance of a query refs the buffer of this query here, should we unref it in `GlobalDriverDispatcher::finalize_driver()` for each fragment instance?